lib/msal-angular/src/msal.interceptor.config.ts

```diff
@@ -7,6 +7,6 @@ import { PopupRequest, RedirectRequest, InteractionType, SilentRequest } from "@
 
 export type MsalInterceptorConfiguration = {
     interactionType: InteractionType.Popup | InteractionType.Redirect;
-    protectedResourceMap: Map<string, Array<string>>;
+    protectedResourceMap: Map<string, Array<string>> | null;
```

Comment:
I think we want the `null` inside the map, like

```suggestion
    protectedResourceMap: Map<string, Array<string> | null>;
```

Reply:
Also, we could make the protectedResourceMap optional?